src/Components/Web.JS/src/Platform/WebView/WebViewIpcReceiver.ts

```diff
@@ -33,6 +33,8 @@ export function startIpcReceiver() {
 
     'EndInvokeDotNet': DotNet.jsCallDispatcher.endInvokeDotNetFromJS,
 
+    'ReceiveByteArray': DotNet.jsCallDispatcher.receiveByteArray,
```

Comment:
WebView still needs to be appropriately tested, this is just a stopgap right now.